db-checkbook/build_scripts/02_build.py

```diff
@@ -1,80 +1,70 @@
 import pandas as pd
 import geopandas as gpd
-import client
 import re
 import os
-from shapely.geometry import Point # testing
 
-"""
-merge_cpdb_geoms: this function is essentially the same as in my jupyter notebook:
-can someone help me think through how to modify it for using the data 
-from digital ocean? I think I'm a little confused as to how exactly this needs to change
-"""
+PATH_DIR = '../.library/'
```

Comment:
sorry, I think I misspoke here - how about `LIB_DIR`?

Reply:
And depending on where you run `python` from, this might break since it's a relative path. 
However, pathlib.Path is awesome - it would allow you to write this as 
```
_curr_file_path = Path(__file__).resolve()
LIB_DIR = _curr_file_path / ".." / ".library" 
```